packages/db/test/unit/controller/level.test.ts

```diff
@@ -113,7 +113,7 @@ describe("LevelDB controller", () => {
     expect(result.length).to.be.equal(2);
   });
 
-  it("test compactRange + approximateSize", async () => {
+  it.skip("test compactRange + approximateSize", async () => {
```

Comment:
Do not unconditionally skip, add
```
if (!linux) this.skip()
```

Reply:
I still think the better approach is not to have platform dependent checks like this...but if you insist I'll add it. In the meantime this PR is in draft and I will make sure not to have the skip checked in when I mark it as ready for review.